Source/RapyutaSimulationPlugins/Public/Robots/RRBaseRobot.h

```diff
@@ -55,14 +71,59 @@ class RAPYUTASIMULATIONPLUGINS_API ARRBaseRobot : public ARRBaseActor
     TSubclassOf<URRRobotROS2Interface> ROS2InterfaceClass;
 
     //! Robot's ROS2 Interface
-    UPROPERTY(Replicated)
+    UPROPERTY(VisibleAnywhere, Replicated, ReplicatedUsing = OnRep_ROS2Interface)
     URRRobotROS2Interface* ROS2Interface = nullptr;
+    
+    UFUNCTION(BlueprintCallable)
+    virtual void OnRep_ROS2Interface();
+
+    /**
+     * @brief Flag to start/stop ROS2Interfaces. Since RPC can't be used, use replication to trigger initialization.
+     */   
+    UPROPERTY(VisibleAnywhere, Replicated, ReplicatedUsing = OnRep_StartStopROS2Interface)
+    bool StartStopROS2Interface = false;

```

Comment:
```suggestion
    bool bStartStopROS2Interface = false;
```
Guess we had this noted various times before for `b` prefix with boolean as to be easily recognized in a check versus a pointer